src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -2971,9 +2971,9 @@ def test_deployment_group_cross_tenant(self, resource_group):
             'image': self.create_random_name('cli_crosstenantimage', 40),
             'version': '1.1.2',
             'captured': self.create_random_name('cli_crosstenantmanagedimage', 40),
-            'aux_sub': '1c638cf4-608f-4ee6-b680-c329e824c3a8',
+            'aux_sub': '6898adc8-5045-473d-a1bf-7012564f43cb',
             'rg': self.create_random_name('cli_test_cross_tenant_rg', 40),
-            'aux_tenant': '72f988bf-86f1-41af-91ab-2d7cd011db47'
+            'aux_tenant': '01a4073e-87c8-47cd-aafc-1439b4b5ea2c'
```

Comment:
Please do not change the `aux_tenant` and `aux_sub` of this test. If you don't have the permission to execute this test, I can rerecord yaml files instead of you

Reply:
Yes this is an issue of permissions.